libs/utils/utils.js

```diff
@@ -1,3 +1,5 @@
+import * as taxonomyLibrary from '../scripts/taxonomy.js';
```

Comment:
This shouldn't be here. Only blocks that need this should be importing it. An entire site should not have to pay the performance penalty of this import.